contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -2,62 +2,155 @@ category: application
 doc: |
   This is the most general application definition for multidimensional
   photoelectron spectroscopy.
+  
+  Groups and fields are named according to the
+  `ISO 18115-1:2023`_ specification as well as the `IUPAC Recommendations 2020`_.
+  
+  .. _ISO 18115-1:2023: https://www.iso.org/standard/74811.html
+  .. _IUPAC Recommendations 2020: https://doi.org/10.1515/pac-2019-0404
+symbols:
+  doc: |
+    The symbols used in the schema to specify e.g. dimensions of arrays
+  n_transmission_function: |
+    Number of data points in the transmission function.
 type: group
 NXmpes(NXobject):
   (NXentry):
+    definition:
+      \@version:
+      enumeration: [NXmpes]
     title:
     start_time(NX_DATE_TIME):
       doc: |
         Datetime of the start of the measurement.
```

Comment:
UTC time stamps and dates wherever possible, maybe hint to it via docstring i.e. ISO8601

Reply:
[NX_DATE_TIME](https://manual.nexusformat.org/nxdl-types.html#index-9) already states it should be a ISO8601 date/time stamp, so I would argue you don't need to specify it here. I have added it nevertheless.

Reply:
Indeed ISO8601 has the "recommendation" to add the time zone but it is only a recommendation, if you where to ask me I would make the time zone required